Double tap fullscreen

paxperscientiam opened the pull request:
Hello @rejas 

This PR is not yet complete; however, I thought you might like to play with it. To try it out, activate ILB and press enter; this will take you into fullscreen mode.

- [x] incorporate fullscreen option
- [x] Press enter for fullscreen.


- [ ] smooth out transition to fullscreen mode. currently ugly
- [ ] double-tap for fullscreen or to exit fullscreen

rejas commented:
Awesome stuff. Only missing I guess is a config option for this ("enableFullscreen" I'd propose) and in the docs. Then I would already merge it (and for a later version think about the possibility to trigger fullscreen via an event :-)

paxperscientiam commented:
Hey, thanks! I actually named the option "fullscreen." Should I change it to "enableFullscreen"?

I added an example to http://localhost:8080/docs/.

I also added a brief explanation on 'README.md".

What about mobile? Timing taps seems a bit complicated -- is it preferred or would you consider a second button?

Do you know if it's possible to smooth out transition to and from fullscreen?

rejas commented:
Merged it as I liked it too much :-) Regarding mobile: That could also be handled with an event that the user of this lib can just call (and hence decide for himself how his users can trigger fullscreen). 
Fullscreen transition smoothnes? I got no idea :-)
